src/mpi/mpi_caf.c

```diff
@@ -49,11 +49,14 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.  */
 
 
 #ifdef GCC_GE_7
+/* This data structure is owned by the library and is treated
+   as a black box by the compiler.
+*/
 typedef struct mpi_caf_token_t
-{
-  void *local_memptr;
-  MPI_Win memptr;
-  MPI_Win *desc;
+{ 
```

Comment:
Sorry.  Please remind me what the appropriate step are to prevent that.  Is there a tool that strips trailing white space?  I'll push a new commit without the trailing white space.